custom_components/waste_collection_schedule/waste_collection_schedule/source/scenicrim_qld_gov_au.py

```diff
@@ -0,0 +1,118 @@
+import csv
+import logging
+from datetime import datetime, timedelta
+
+import requests
+from dateutil.rrule import FR, MO, TH, TU, WE, WEEKLY, rrule
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+
+TITLE = "Scenic Rim Regional Council"
+DESCRIPTION = "Source for scenicrim.qld.gov.au services for Scenic Rim Regional Council"
+URL = "https://scenicrim.qld.gov.au"
+TEST_CASES = {
+    "Red Week": {
+        "address": "The Old Avocado Farm 77A Long Road TAMBORINE MOUNTAIN  QLD 4272",
+    },
+    "Blue Week": {
+        "address": "Elysian Fields 2/3043 Beaudesert-Nerang Road WONGLEPONG  QLD 4275"
+    },
+}
+API_URL = "https://srrcwastebinserviceday.blob.core.windows.net/wastebinservicedayexport/WasteBinServiceDay_SRRCWebsiteSearch.csv"
+ICON_MAP = {
+    "GENERAL WASTE": "mdi:trash-can",
+    "RECYCLING": "mdi:recycle",
+}
+DAYS: dict = {
+    "MONDAY": MO,
+    "TUESDAY": TU,
+    "WEDNESDAY": WE,
+    "THURSDAY": TH,
+    "FRIDAY": FR,
+}
+WEEKDAYS: list = [MO, TU, WE, TH, FR]
+START_DATES: dict = {  # taken from https://www.scenicrim.qld.gov.au/downloads/file/6551/your-waste-bins-and-facilities-guide
+    "BLUE": datetime(2024, 12, 2, 0, 0, 0),  # known Monday in a Red Week
+    "RED": datetime(2024, 12, 9, 0, 0, 0),  # known Monday in a Blue Week
+}
+NOW: datetime = datetime.now()
+START_DATE: datetime = NOW + timedelta(days=-1)
+END_DATE: datetime = NOW + timedelta(days=14)
```

Comment:
I don't think this is a good idea. This is executed once when loading this module and does not every fetch, so END_DATE will be in the past if the HA instance runs 14 days

Reply:
Good point. Now moved to _fetch_.